src/Plugins/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -212,81 +212,122 @@ private static Witness[] WitnessesFromJson(JArray _params)
             }).ToArray();
         }
 
-        [RpcMethod]
-        protected internal virtual JToken InvokeFunction(JArray _params)
+        /// <summary>
+        /// Invokes a smart contract with its scripthash based on the specified operation and parameters and returns the result.
+        /// </summary>
+        /// <remarks>
+        /// This method is used to test your VM script as if they ran on the blockchain at that point in time.
+        /// This RPC call does not affect the blockchain in any way.
+        /// </remarks>
+        /// <param name="scriptHash">Smart contract scripthash. Use big endian for Hash160, little endian for ByteArray.</param>
```

Comment:
> Use big endian for Hash160

It's LE.

Reply:
Not resolved.

Reply:
Can you check it again, cause its from the official document.